facebook4j.properties

```diff
@@ -0,0 +1,3 @@
+debug=true
+oauth.appId=131555220900267
+oauth.appSecret=31851163b721d5e249b58b44bef17246
```

Comment:
Not really relevant here I guess, but for future reference OAuth credentials should not be committed to github, otherwise many bad things man happen :<